draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -542,6 +554,19 @@ use an approach such as {{!HappyEyeballsV2=RFC8305}}.
 Some important optimizations are discussed in {{optimizations}}
 to avoid additional latency in comparison to ordinary AAAA/A lookups.
 
+## Clients using a Proxy
+
+Clients using a proxy via issuing CONNECT requests
+({{!RFC7231}} Section 4.3.6) should follow the same
+resolution for resolving SVCB records.  If a ServiceForm SVCB
```

Comment:
It sounds like you are proposing to chase the SVCB chain outside the proxy, and then use the final SvcDomainName in the request-target.  That seems like a fine idea to me, but I think the recommendation should be weaker.  I don't want anyone to think that we're telling them to leak DNS queries to an untrusted resolver when using a proxy for privacy purposes.